tutorials/smart-contracts/deploy-erc20.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Deploy a ERC20 to Asset Hub
+description: Deploy an ERC20 token on Asset Hub using PolkaVM. This guide covers contract creation, compilation, deployment, and interaction via Polkadot Remix IDE.
+---
+
+# Deploy ERC20 to Asset Hub
+
+## Introduction
+
+ERC20 tokens are fungible tokens commonly used for creating cryptocurrencies, governance tokens, and staking mechanisms. Asset Hub enables easy token deployment with EVM-compatible smart contracts via PolkaVM. This tutorial covers deploying an ERC20 contract on the Westend TestNet using [Polkadot Remix IDE](https://polkadot.remix.io){target=\_blank}, a web-based development tool. [OpenZeppelin's ERC20 contracts](https://github.com/OpenZeppelin/openzeppelin-contracts/tree/v5.0.0/contracts/token/ERC20){target=\_blank} is used for security and compliance.
+
+## Prerequisites
+
+Before starting, make sure you have:
+
+- MetaMask installed and connected to Westend Asset Hub
+- A funded account with some WND tokens (you can get them from the [Westend Faucet](https://faucet.polkadot.io/westend?parachain=1000){target=\_blank})
+- Basic understanding of Solidity and fungible tokens
+
+## Creating the ERC20 Contract
+
+To create the ERC20 contract, you can follow the steps below:
+
+1. Navigate to the [Polkadot Remix IDE](https://polkadot.remix.io){target=\_blank}
+2. Click in the **Create new file** button under the **contracts** folder, and name your contract as `MyToken.sol`
+
+    ![](/images/tutorials/smart-contracts/deploy-erc20/deploy-erc20-1.webp)
+
+3. Now, paste the following ERC20 contract code into the editor
+
+    ```solidity title="MyToken.sol"
+    --8<-- 'code/tutorials/smart-contracts/deploy-erc20/MyToken.sol'
+    ```
+
+    The key components of the code above are:
+
+    - Contract Imports
+
+        - [`ERC20.sol`](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v5.0.0/contracts/token/ERC20/ERC20.sol){target=\_blank} - the base contract for fungible tokens, implementing core functionality like transfers, approvals, and balance tracking
+        - [`Ownable.sol`](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v5.0.0/contracts/access/Ownable.sol){target=\_blank} - provides basic authorization control, ensuring only the contract owner can mint new tokens
+    
+    - Constructor Parameters
+
+        - `initialOwner` - sets the address that will have administrative rights over the contract
+        - `"MyToken"` - the full name of your token
+        - `"MTK"` - the symbol representing your token in wallets and exchanges
+
+    - Key Function
```

Comment:
Even though the only visible funciton in this code is `mint`, you should explain that there are other functions available such as `transfer`, `approve`, etc